Containerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/operate-first/opf-toolbox:v0.4.2 AS toolbox
+FROM quay.io/thoth-station/s2i-custom-py38-notebook:v0.3.2
+
+ENV XDG_DATA_HOME=/usr/share/.local/share \
+          XDG_CACHE_HOME=/usr/share/.cache \
+          XDG_CONFIG_HOME=/usr/share/.config
```

Comment:
Nit: I have a slight formatting issue with this. Let's align this and use 4 spaces.

Reply:
Fixed and ready to be shiped :rocket: